_infra/helm/response-operations-ui/Chart.yaml

```
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.5.41
+version: 2.5.42
```

```suggestion
version: 2.6.0
```

This is big enough for this

---

👍